pkg/virt-handler/vm.go

```diff
@@ -2721,6 +2722,22 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 		}
 	}
 
+	isolationRes, err := d.podIsolationDetector.Detect(vmi)
+	if err != nil {
+		return fmt.Errorf(failedDetectIsolationFmt, err)
+	}
+
+	// Backwards compatibility
```

Comment:
This code needs to be moved to the relevant part. Follow `FormatDownwardMetricPath`